app/models/scorebook.rb

```diff
@@ -0,0 +1,97 @@
+class Scorebook
```

Comment:
@marcellosachs That could work, I feel like it's basically a model, though not an ActiveRecord model. I'd be inclined to leave it but maybe change the name to something like `ScorebookQuery`?